conf/agent.conf.template

```diff
@@ -147,6 +147,25 @@ cygnusagent.sinks.mongo-sink.db_prefix =
 # prefix pro the MongoDB collections (empty for none)
 cygnusagent.sinks.mongo-sink.collection_prefix =
 
+# ============================================
+# OrionSTHSink configuration
+# sink class, must not be changed
+cygnusagent.sinks.sth-sink.type = com.telefonica.iot.cygnus.sinks.OrionSTHSink
+# channel name from where to read notification events
+cygnusagent.sinks.sth-sink.channel = sth-channel
+# FQDN/IP:port where the MongoDB server runs (standalone case) or comma-separated list of FQDN/IP:port pairs where the MongoDB replica set members run
+cygnusagent.sinks.sth-sink.mongo_hosts = x1.y1.z1.w1:port1,x2.y2.z2.w2:port2,...
+# a valid user in the MongoDB server (or empty if authentication is not enabled in MongoDB)
+cygnusagent.sinks.sth-sink.mongo_username = mongo_username
+# password for the user above (or empty if authentication is not enabled in MongoDB)
+cygnusagent.sinks.sth-sink.mongo_password = xxxxxxxx
+# data model (collection-per-service-path, collection-per-entity, collection-per-attribute)
+cygnusagent.sinks.sth-sink.data_model = collection-per-entity
+# prefix for the MongoDB databases (empty for none)
+cygnusagent.sinks.sth-sink.db_prefix =
```

Comment:
Although this is just a template, I guess many users will try to use it as it is without modifying it :) Since the MongoDB instances could be reused by distinct components, I would suggest to include some kind of prefix as the default value instead of empty. Something such as "sth_" or whatever to avoid database name collisions.